File: app/src/main/java/com/adisa/diningplus/db/dao/ManagerDao.java
package com.adisa.diningplus.db.dao;

import androidx.room.Dao;
import androidx.room.Delete;
import androidx.room.Insert;
import androidx.room.Query;
import androidx.room.Update;

import com.adisa.diningplus.db.entities.Manager;

import java.util.List;

@Dao
public interface ManagerDao {

    @Query("SELECT * FROM managers")
    List<Manager> getAll();

    @Insert
    void insert(Manager manager);

    @Delete
    void delete(Manager manager);

    @Update
    void update(Manager manager);
}

File: app/src/main/java/com/adisa/diningplus/db/dao/ManagerDao.kt
package com.adisa.diningplus.db.dao

import androidx.room.*
import com.adisa.diningplus.db.entities.Manager

@Dao
interface ManagerDao {
    @Query("SELECT * FROM managers")
    fun getAll(): List<Manager?>

    @Insert
    fun insert(manager: Manager?)

    @Delete
    fun delete(manager: Manager?)

    @Update
    fun update(manager: Manager?)
}